docs/QUICKSTART.md

```diff
@@ -1,76 +1,195 @@
-- Install eraser from [deployment YAML](https://github.com/Azure/eraser/blob/v0.1.0/deploy/eraser.yaml).
-
-```bash
-$ kubectl apply -f https://raw.githubusercontent.com/Azure/eraser/v0.1.0/deploy/eraser.yaml
-namespace/eraser-system created
-customresourcedefinition.apiextensions.k8s.io/imagejobs.eraser.sh created
-customresourcedefinition.apiextensions.k8s.io/imagelists.eraser.sh created
-serviceaccount/eraser-controller-manager created
-role.rbac.authorization.k8s.io/eraser-leader-election-role created
-clusterrole.rbac.authorization.k8s.io/eraser-manager-role created
-rolebinding.rbac.authorization.k8s.io/eraser-leader-election-rolebinding created
-clusterrolebinding.rbac.authorization.k8s.io/eraser-manager-rolebinding created
-configmap/eraser-manager-config created
-deployment.apps/eraser-controller-manager created
-
-$ kubectl get pods -n eraser-system
-NAME                                         READY   STATUS    RESTARTS   AGE
-eraser-controller-manager-759c7c8794-q8qqg   1/1     Running   0          22s
-```
-
-- Create an [ImageList](./test/e2e/test-data/eraser_v1alpha1_imagelist.yaml) and specify the images you would like to remove.
-
-Example:
-```bash
-cat <<EOF | kubectl apply -f -
-apiVersion: eraser.sh/v1alpha1
-kind: ImageList
-metadata:
-  name: imagelist
-spec:
-  images:
-    - sha256:2834dc507516af02784808c5f48b7cbe38b8ed5d0f4837f16e78d00deb7e7767
-    - docker.io/library/nginx:latest
-    - redis
-EOF
-```
+# Quickstart Tutorial
+
+This tutorial demonstrates the functionality of Eraser and validates that non-running images are removed after applying an `ImageList` with the declared images.
+
+1. Create a cluster
+
+  Eraser can be deployed to any Kubernetes cluster, however, for the purposes of this tutorial a `Kind` cluster will be created with a control-plane and two worker nodes.
+
+    ``` shell
+    # get kind
+    curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.13.0/kind-linux-amd64
```

Comment:
let's link to kind docs for this since users might not be on linux/amd64 or might want to use a pkg mgr etc
https://kind.sigs.k8s.io/docs/user/quick-start/